src/components/railBottom/controls/Controls.vue

```diff
@@ -1,195 +1,145 @@
-          label: this.$t('Form Task'),
-          id: 'task',
-          items: [
-            {
-              iconSrc: require('@/assets/toolpanel/task.svg'),
-              label: this.$t('Form Task'),
-              id: 'formTask',
-            },
-            {
-              iconSrc: require('@/assets/toolpanel/manual-task.svg'),
-              label: this.$t('Manual Task'),
-              id: 'manualTask',
-            },
-            {
-              iconSrc: require('@/assets/toolpanel/script-task.svg'),
-              label: this.$t('Script Task'),
-              id: 'script',
-            },
-            {
-              iconSrc: require('@/assets/toolpanel/task.svg'),
-              label: this.$t('Sub Process'),
-              id: 'subProcess',
-            },
-          ],
-        },
-        {
-          iconSrc: require('@/assets/toolpanel/generic-gateway.svg'),
-          label: this.$t('Gateway'),
-          id: 'gateway',
-          items: [
-            {
-              iconSrc: require('@/assets/toolpanel/exclusive-gateway.svg'),
-              label: this.$t('Exclusive Gateway'),
-              id: 'exclusiveGateway',
-            },
-            {
-              iconSrc: require('@/assets/toolpanel/inclusive-gateway.svg'),
-              label: this.$t('Inclusive Gateway'),
-              id: 'inclusiveGateway',
-            },
-            {
-              iconSrc: require('@/assets/toolpanel/parallel-gateway.svg'),
-              label: this.$t('Parallel Gateway'),
-              id: 'parallelGateway',
-            },
-            {
-              iconSrc: require('@/assets/toolpanel/generic-gateway.svg'),
-              label: this.$t('Event Based Gateway'),
-              id: 'eventBasedGateway',
-            },
-
-          ],
-        },
-
-        {
-          iconSrc: require('@/assets/toolpanel/pool.svg'),
-          label: this.$t('Pool'),
-          id: 'pool',
-        },
-      ],
-      plusIcon: require('@/assets/railBottom/plus-lg-light.svg'),
+      plusIcon: require('@/assets/railBottom/plus.svg'),
+      wasClicked: false,
+      element: null,
+      selectedItem: null,
+      selectedSubmenuItem: null,
+      xOffset: 0,
+      yOffset: 0,
+      movedElement: null,
+      isDragging: false,
+      helperStyles: {
+        backgroundColor:'#ffffff',
+        position: 'absolute',
+        height: '40px',
+        width: '40px',
+        zIndex: '10',
+        opacity: '0.5',
+        pointerEvents: 'none',
+      },
+      popperType: null,
     };
   },
+  computed: {
+    controls() {
+      return this.nodeTypes
+        .filter(nodeType => nodeType.control)
+        .map(nodeType => ({
+          type: nodeType.id,
+          icon: nodeType.icon,
+          label: nodeType.label,
+          bpmnType: nodeType.bpmnType,
+          rank: nodeType.rank || BOTTOM,
+          items: nodeType.items || null,
+        }))
+        .sort((node1, node2) => node1.rank - node2.rank);
+    },
```

Comment:
this probably can be fetched from the Vuex store we created to get all the node types

Reply:
It will be applied